### PR TITLE
(chore:1680) - Updated the test cases for the Search Microservice

### PR DESCRIPTION
**Pull request by riginoommen**
# Resolves 1680

# Explain the change
Updated the test cases with respective to new APIs

## Does this PR introduce a breaking change

No

## Screenshots

<details>
<summary>View Screenshots</summary>
image
</details>

### Ready-for-merge Checklist

- [x] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [x] Does the change have appropriate unit tests?
- [x] Did tests pass?

### CODE REVIEW

File: packages/search-service/src/e2e/search.spec.ts [hunk 1]
```diff
@@ -1,16 +1,44 @@
-import dotenv from 'dotenv';
+import dotenv from "dotenv";
```
Comment (deshmukhmayur) [inline packages/search-service/src/e2e/search.spec.ts:1]: Can you change all the double-quotes to single? To make it more consistent with the rest of the formatting in the repo.
Also make sure your editor uses the vscode settings specified in the `.vscode/settings.json` file at the root of the repo. It already has all the necessary formatting rules mentioned.

**Review by deshmukhmayur — CHANGES_REQUESTED** [review]

File: packages/search-service/src/e2e/search.spec.ts (hunk 1 above)
Comment (riginoommen) [reply packages/search-service/src/e2e/search.spec.ts]: Updated

**Review by deshmukhmayur — APPROVED** [review]
LGTM.

**Review by akhilmhdh — APPROVED** [review]
LGTM

**Pull request merged**